lbrynet/daemon/Daemon.py

```diff
@@ -1199,6 +1207,67 @@ def jsonrpc_wallet_balance(self, address=None, include_unconfirmed=False):
                 self.session.wallet.get_address_balance(address, include_unconfirmed)))
 
     @defer.inlineCallbacks
+    def jsonrpc_wallet_unlock(self, password):
+        """
+        Unlock an encrypted wallet
+
+        Usage:
+            wallet_unlock (<password>)
+
+        Returns:
+            (bool) true if wallet is unlocked, otherwise false
+        """
+
+        if not self.session.wallet._cmd_runner.locked:
```

Comment:
isn't it bad to access "private" members?

also, isn't `_cmd_runner` sometimes `None`?